lib/active_merchant/billing/gateways/paypal_express_common.rb

```diff
@@ -16,13 +16,11 @@ def redirect_url
         test? ? test_redirect_url : live_redirect_url
       end
 
-      def redirect_url_for(token, options = {})
+      def base_redirect_url_for(token, options: {}, cmd: nil)
         options = {:review => true, :mobile => false}.update(options)
```

Comment:
@Krystosterone changing to an options named parameter will break all existing implementations :/, I do agree it's weird to have both.